Templates/Tests/Generated/AutoEncodable.generated.swift

```diff
@@ -0,0 +1,35 @@
+// Generated using Sourcery 0.10.1 — https://github.com/krzysztofzablocki/Sourcery
+// DO NOT EDIT
+
+
+
+
+
+
+// MARK: AutoDecodableTest+Encodable | AutoEncodable
+extension AutoDecodableTest {
+  private enum EncodableKeys: String, CodingKey {
+    case bool  = "auto_codable_bool"
+    case optional 
+    case prim1  = "auto_codable_prim_1"
+    case prim2  = "auto_codable_prim_2"
+    case prim3  = "auto_codable_prim_3"
+    case arr 
+    case arrOpt 
+    case date 
+    case optDate 
+  }
+
+  internal func encode(to encoder: Encoder) throws {
+    var container = encoder.container(keyedBy: EncodableKeys.self)
+    try container.encode(bool ? 1 : 0, forKey: .bool)
```

Comment:
this will encode Int, wo it will not be decoded as Bool, I supposed, so this should be just `bool`. The case when API expects 1/0 instead of true/false should be handled manually.